api/v1/db/oracledb/setupDB.sql

```diff
@@ -0,0 +1,76 @@
+DROP TABLE PARKS;
+DROP TABLE OWNERS;
+
+CREATE TABLE PARKS (
+	ID NUMBER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1) NOT NULL,
+	NAME VARCHAR2(225) NOT NULL,
+	STREET_ADDRESS VARCHAR2(225) NOT NULL,
+	CITY VARCHAR2(225) NOT NULL,
+	STATE VARCHAR2(225) NOT NULL,
```

Comment:
If state should only be 2 characters, you could use the `CHAR` type with length 2